### PR TITLE
OCM-4931: Remove a validation in machinepool so it can be created with the subnet from different AZ than the cluster was created - for singleAZ cluster

### PR DESCRIPTION
**Pull request by bardielle**
@amalykhi

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 1.4.1 (Nov 20, 2023)
```
Comment (gdbranco) [inline CHANGELOG.md:1]: Extra change I believe

Comment (bardielle) [reply CHANGELOG.md:1]: This is the change log for official release version, it's a cherry-pick from v1.4.1 
we will release that bug fix int he next release I guess

**Pull request merged**